examples/plot_localizer_analysis_permutation.py

```diff
@@ -0,0 +1,148 @@
+
+# Specify the contrasts
+contrast_matrix = np.eye(design_matrix.shape[1])
+contrasts = dict([(column, contrast_matrix[i])
+                  for i, column in enumerate(design_matrix.columns)])
+
+contrasts["audio"] = contrasts["clicDaudio"] + contrasts["clicGaudio"] +\
+    contrasts["calculaudio"] + contrasts["phraseaudio"]
+contrasts["video"] = contrasts["clicDvideo"] + contrasts["clicGvideo"] + \
+    contrasts["calculvideo"] + contrasts["phrasevideo"]
+contrasts["computation"] = contrasts["calculaudio"] + contrasts["calculvideo"]
+contrasts["sentences"] = contrasts["phraseaudio"] + contrasts["phrasevideo"]
+
+# Short list or more relevant contrasts
+# contrasts = {
+#     "left-right": (contrasts["clicGaudio"] + contrasts["clicGvideo"]
+#                    - contrasts["clicDaudio"] - contrasts["clicDvideo"]),
+#     "H-V": contrasts["damier_H"] - contrasts["damier_V"],
+#     "audio-video": contrasts["audio"] - contrasts["video"],
+#     "video-audio": -contrasts["audio"] + contrasts["video"],
+#     "computation-sentences": (contrasts["computation"] -
+#                               contrasts["sentences"]),
+#     "reading-visual": contrasts["phrasevideo"] - contrasts["damier_H"]
+#     }
+
+contrasts = {
+    "reading-visual": contrasts["phrasevideo"] - contrasts["damier_H"]
+    }
+
+### Permutation tests ###########################################
+
+# Design matrix obj arrangement for paradigm permutations
+# What about creating design_matrix class to wrap make design matrix and
+# respective parameters. Then could just pass that object
+
+design_matrix_obj = [frame_times, paradigm,
+                     {'hrf_model': 'canonical with derivative',
+                      'drift_model': 'cosine',
+                      'period_cut': 128}]
+
+# Run the permutations
+
+res = first_level_permutation_test(contrasts, fmri_glm, fmri_img,
+                                   [design_matrix_obj], n_perm=600,
+                                   two_sided_test=True,
+                                   cluster_threshold=None,
+                                   random_state=None, verbose=10,
+                                   n_jobs=n_proc)
+
+### Plots #######################################################
+
+# Create snapshots of the contrasts thresholded with parametric and non
+# parametric methods.
+
+# write directory
+write_dir = 'results'
+if not path.exists(write_dir):
+    mkdir(write_dir)
+
+for index, (contrast_id, (unc_pval, cor_pval)) in enumerate(res.iteritems()):
+    z_map, = fmri_glm.transform(contrasts[contrast_id],
+                                contrast_name=contrast_id,
+                                output_z=True)
+    nib.save(z_map, path.join(write_dir, '%s_z_map.nii' % contrast_id))
+    z_map_data = fmri_glm.masker_.transform(z_map)[0]
+
+    display = plotting.plot_stat_map(z_map, display_mode='z',
+                                     threshold=z_score(0.1),
```

Comment:
you're a bit extreme in the choice of the threshold. For an uncorrected  p-value, rather take .001